llvm/test/Transforms/InstCombine/cast-call-naked.ll

```diff
@@ -0,0 +1,16 @@
+; RUN: opt < %s -passes=instcombine -S | FileCheck %s
+
+define dso_local void @naked_func() #0 {
```

Comment:
Remove dso_local.

Also directly use `naked` here instead of an attribute group.